src/View/StringTemplate.php

```diff
@@ -284,7 +306,7 @@ protected function _formatAttribute($key, $value, $escape = true)
         $truthy = [1, '1', true, 'true', $key];
         $isMinimized = isset($this->_compactAttributes[$key]);
         if ($isMinimized && in_array($value, $truthy, true)) {
-            return "$key=\"$key\"";
+            return "$key";
```

Comment:
Yes, you are right. Not necessary very much.

I didn't know those two formats are the same and it took me long
how I could implement the shorten version of attributes in cake.

I found that the doc comments in the source say that you can make 
it compact, and this is another reason why I report it.

Now I know this is not a necessary change at all. Thank you for your care.